rewards/snapshot/claims_snapshot.py

```diff
@@ -6,14 +6,16 @@
 from badger_api.requests import fetch_all_claimable_balances
 from helpers.constants import CLAIMABLE_TOKENS, DIGG
 from helpers.digg_utils import digg_utils
-from helpers.enums import BalanceType
+from helpers.enums import BalanceType, Network
 from helpers.web3_utils import make_token
 from rewards.classes.Snapshot import Snapshot
 
 
 @lru_cache
 def claims_snapshot(chain: str) -> Dict[str, Snapshot]:
     all_claims = fetch_all_claimable_balances(chain)
+    if not all_claims:
+        raise ValueError("Badger-api returns error")
```

Comment:
Do we want to be more descriptive with this error?  Maybe `Error retrieving claimable balances from Badger API`?

Reply:
Sure thing

Reply:
Fixed